### ISSUE TITLE
Refactoring the DebugReportDialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#231]
```
Go in the About / Debugging information , click on send to dev team
Now check to bugreport on the wiki site

    http://www.freemedforms.com/en/bugreport?do=index

there should be a new bug report but it does not appear.

Problem can be:
- in the C++ code
- in the PHP code
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 4 Dec 2012 at 10:15

**Comment by GoogleCodeExporter** [comment]
Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 5:50
- Changed title: **Refactoring the DebugReportDialog**
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 3f8450d7ab81.

Moving debugreportdialog to Utils, making a more generic dialog
```

Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 8:06

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f4fb54dabf1c.
```

Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 8:06
- Changed state: **Fixed**
